cmd/storagenode/main.go

```diff
@@ -161,6 +161,11 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 		return errs.New("Error creating tables for master database on storagenode: %+v", err)
 	}
 
+	err = peer.Storage2.Store.InitCache()
```

Comment:
I think we may want to avoid blocking startup here while we wait for a full dir iteration. Maybe this call should just kick off a fresh calculation instead of blocking, or (my vote) we should persist the most recent full count to disk and load it here as our space-used starting point.

Reply:
i changed it to this based on a comment from egon https://github.com/storj/storj/pull/2707#discussion_r310642467.

But yeah blocking on startup could take a long time if there is a lot stored, so maybe this isn't ideal.  A fresh calculation could work except that we would either need to halt access to the cache value until the calculation is complete or be ok with is being a zero value and potentially very inaccurate.

So maybe persisting the most recent count is ideal. I will look into adding that.

Reply:
Yeah, we want to avoid blocking startup as much as possible. The question is what is the implication of not having up to date information about this.